crates/irust/src/dependencies.rs

```diff
@@ -38,47 +38,7 @@ pub fn check_required_deps() -> bool {
 }
 
 pub fn warn_about_opt_deps(options: &mut Options) {
-    let opt_deps: [Dep; 4] = [
```

Comment:
With new rustup 1.26 (released today) there is no need to manually add rust-analyzer to path since they are placing proxy rust-analyzer in `.cargo/bin`.
I would suggest to add `rustup --version` check here and if its smaller than 1.26.0 than ask to do `rustup self update`.
And after that  `rustup component add rust-analyzer` could be invoked and it will be in path.

https://github.com/rust-lang/rustup/blob/stable/CHANGELOG.md

Reply:
Thanks I'll keep it in mind , I expect though that anyone who are using rust already have rust-analyzer installed so its fine for now, I can add this in a later commit

Reply:
I didn't have it for example because I used only VSCode which uses its own version of rust-analyzer